windows-msi/msi.wxs

```diff
@@ -49,7 +49,7 @@
             SourceFile="artwork\tapctl_create.ico"/>
         <Binary
             Id="libopenvpnmsica.dll"
-            SourceFile="!(bindpath.openvpnbin)libopenvpnmsica.dll"/>
+            SourceFile="!(bindpath.openvpnmsica)openvpnmsica.dll"/>
```

Comment:
Since we got rid of "lib" prefix we might want to remove it from "Id" as well, but this could be done later.